src/main/java/uk/gov/hmcts/reform/ccd/client/CaseAccessApi.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.ccd.client;
+
+import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import uk.gov.hmcts.reform.ccd.client.model.UserId;
+
+import java.util.List;
+
+@FeignClient(name = "ccd-access-api", url = "${core_case_data.api.url}",
+        configuration = CoreCaseDataConfiguration.class)
+public interface CaseAccessApi {
```

Comment:
What do you think about `CaseAuthorisationApi`? At first I thought this class will be about retrieving cases.